letsencrypt-win-simple/Plugin/ManualPlugin.cs

```diff
@@ -107,7 +107,14 @@ public override void HandleMenuResponse(string response, List<Target> targets)
                 if (Program.Options.San)
                 {
                     Console.Write("Enter all Alternative Names seperated by a comma ");
-                    Console.SetIn(new System.IO.StreamReader(Console.OpenStandardInput(8192)));
+
+                    //Console.SetIn(new System.IO.StreamReader(Console.OpenStandardInput(8192)));
```

Comment:
If you check your code into the same branch you did this PR from, it should merge it right into this same PR. Here's some more info on [stackoverflow](http://stackoverflow.com/questions/7947322/preferred-github-workflow-for-updating-a-pull-request-after-code-review).